maths/palindromic_number_checker.py

```diff
@@ -0,0 +1,24 @@
+def is_number_palindromic(num: int) -> bool:
+    """
+    This function checks if given number is palindromic or not.
+    >>> is_number_palindromic(121)
+    True
+    >>> is_number_palindromic(1231)
+    False
+    >>> is_number_palindromic(12321)
+    True
+    >>> is_number_palindromic(1211)
+    False
+    """
+    num_rev = str(num)[::-1]
+    num_rev = int(num_rev)
+    if num == num_rev:
+        return True
+    else:
+        return False
```

Comment:
you can just return `num == num_rev`

Reply:
Sure, that is also possible. But I have coded this keeping in mind the fresh developers of Python, That they can grasp it while with a single look.